config/mongoid.yml

```diff
@@ -6,7 +6,7 @@ common: &default_client
       mode: :<%= ENV['MONGOID_READ_MODE'] || 'primary' %>
     max_retries: <%= ENV['MONGOID_MAX_RETRIES'] || 1 %>
     retry_interval: <%= ENV['MONGOID_RETRY_INTERVAL'] || 0 %>
-    timeout: <%= ENV['MONGOID_TIMEOUT'] || 0.5 %>
+    connect_timeout: <%= ENV['MONGOID_TIMEOUT'] || 0.5 %>
```

Comment:
Can we change the name of the environment variable to `MONGOID_CONNECT_TIMEOUT` as well?

I see mongoid has other timeout related options (`socket_timeout`, `server_selection_timeout`, and `wait_queue_timeout`) so a more specific name would help avoid confusion.

Reply:
Yes makes sense, but I suppsoe this would be a breaking change for downstream app(e.g. tutor-forum, edx-configuration), I can hanlde the case for tutor-forum since I maintain it, but sure about edx-configuration. 

But in many case, in practice I don't think it matter that much since the variable was unusable in the first place.